Framework/Algorithms/src/MonteCarloAbsorption.cpp

```diff
@@ -184,381 +206,57 @@ void MonteCarloAbsorption::exec() {
       } else {
         // elastic case already initialized
       }
-      doSimulation(detPos, lambdaIn, lambdaOut, yValues[bin], eValues[bin]);
+      std::tie(signal[j], std::ignore) =
```

Comment:
Is this intentionally throwing away those errors?
I see that the unit test checks the 1/N^2 errors returned from `MCAbsorptionStrategy::calculate()`, but with this the algorithm never returns those values, just zeros?